### PR TITLE
Add acos and asin benchmarks.

### PR DESCRIPTION
**Pull request by pkofod**

### CODE REVIEW

File: src/scalar/ScalarBenchmarks.jl [hunk 1]
```diff
@@ -445,4 +446,45 @@ for T in (Float64, )# (Float32, Float64) add Float32 later
...
+########
+# asin #
+########
+
+g = addgroup!(SUITE, "asin")
+for T in (Float32, Float64)
+    _arg_string = arg_string(T)
+    g["zero", _arg_string] = @benchmarkable asin($(zero(T)))
+    g["small", "positive argument", _arg_string] = @benchmarkable asin($(nextfloat(zero(T))))
+    g["small", "negative argument", _arg_string] = @benchmarkable asin($(nextfloat(zero(T))))
```
Comment (Sacha0) [inline src/scalar/ScalarBenchmarks.jl]: `prevfloat`?

File: src/scalar/ScalarBenchmarks.jl [hunk 2]
```diff
@@ -445,4 +446,45 @@ for T in (Float64, )# (Float32, Float64) add Float32 later
...
+########
+# acos #
+########
+
+g = addgroup!(SUITE, "acos")
+for T in (Float32, Float64)
+    _arg_string = arg_string(T)
+    g["zero", _arg_string] = @benchmarkable acos($(zero(T)))
+    g["small", "positive argument", _arg_string] = @benchmarkable acos($(nextfloat(zero(T))))
+    g["small", "negative argument", _arg_string] = @benchmarkable acos($(nextfloat(zero(T))))
```
Comment (Sacha0) [inline src/scalar/ScalarBenchmarks.jl]: `prevfloat`?

Comment (pkofod) [inline src/scalar/ScalarBenchmarks.jl]: :+1:

Comment (pkofod) [inline src/scalar/ScalarBenchmarks.jl]: Or is that -0.0?

Comment (Sacha0) [inline src/scalar/ScalarBenchmarks.jl]: `prevfloat` seems to do the trick? :)
```julia
julia> nextfloat(zero(Float64))
5.0e-324

julia> prevfloat(zero(Float64))
-5.0e-324
```

**Comment by pkofod** [comment]
If it looks good to you, I don't have anything further I want to add.

**Comment by jrevels** [comment]
These will probably have to have their noise tolerances raised later, but might as well wait until there's a need for it before we do so.

**Comment by pkofod** [comment]
> These will probably have to have their noise tolerances raised later, but might as well wait until there's a need for it before we do so.

That's probably true, probably also for the other benchmarks I added! Is it something I should do during the PR-phase?

**Comment by jrevels** [comment]
> That's probably true, probably also for the other benchmarks I added! Is it something I should do during the PR-phase?

Nope, you're good. It's better to adjust the noise tolerances only if they end up being a problem in practice. Otherwise, we'd be decreasing Nanosoldier's ability to catch regressions for no reason.

**Comment by tkelman** [comment]
The `|` are messing with the markdown formatting again

**Pull request merged**